src/maintagent/agent/usage.c

```diff
@@ -42,6 +42,7 @@ FUNCTION void Usage(char *Name)
   printf("  -T   :: Remove orphaned temp tables.\n");
   printf("  -U   :: Process expired uploads (slow).\n");
   printf("  -Z   :: Remove orphaned files from the repository (slow).\n");
+  printf("  -E   :: Remove orphaned rows from database.\n");
```

Comment:
Can we have a `(slow)` notice like other options? Because this flag executes several large queries.

Reply:
I'm still wondering that it should be kept in the slow list or not. because after we fixed delagent it will not find orphaned files to delete 🤔